docs/deploy-rbd.md

```diff
@@ -55,7 +55,9 @@ make image-cephcsi
 | `dataPool`                                                                                          | no                   | Ceph pool used for the data of the RBD images.                                                                                                                                                                          |
 | `volumeNamePrefix`                                                                                  | no                   | Prefix to use for naming RBD images (defaults to `csi-vol-`).                                                                                                                                                           |
 | `snapshotNamePrefix`                                                                                | no                   | Prefix to use for naming RBD snapshot images (defaults to `csi-snap-`).                                                                                                                                                 |
-| `imageFeatures`                                                                                     | no                   | RBD image features. CSI RBD currently supports only `layering` feature. See [man pages](http://docs.ceph.com/docs/nautilus/man/8/rbd/#cmdoption-rbd-image-feature)                                                         |
+| `imageFeatures`                                                                                     | no                   | RBD image features. CSI RBD currently supports only `layering` feature. See [man pages](http://docs.ceph.com/docs/nautilus/man/8/rbd/#cmdoption-rbd-image-feature)                                                      |
+| `mapOptions`                                                                                        | no                   | Map options to use when mapping rbd image. See [man pages](https://docs.ceph.com/docs/master/man/8/rbd/#kernel-rbd-krbd-options).                                                                                       |
```

Comment:
Is https://docs.ceph.com/docs/master/man/8/rbd-nbd/#options not possible too? Best to mention that the options depend on the `mounter`

Reply:
User has to understand that he should use nbd options when he is using nbd driver and use krbd options when  he is using krbd driver

Reply:
yes, providing links to both sets of options would be good for that reason